packages/application-generic/src/services/queues/queue-base.service.ts

```diff
@@ -66,56 +86,64 @@ export class QueueBaseService {
     );
   }
 
-  public async addMinimalJob(
-    id: string,
-    data?: any,
-    groupId?: string,
-    options: JobsOptions = {}
-  ) {
-    const jobData = data
+  public async addMinimalJob(params: IJobParams) {
+    const { name, groupId, data, options } = params;
+
+    const jobData: IJobData = data
       ? {
           _environmentId: data._environmentId,
-          _id: id,
+          _id: data._id,
           _organizationId: data._organizationId,
           _userId: data._userId,
         }
       : undefined;
 
-    await this.add(id, jobData, groupId, {
-      removeOnComplete: true,
-      removeOnFail: true,
-      ...options,
-    });
+    await this.instance.add(
+      name,
+      jobData,
+      {
+        removeOnComplete: true,
+        removeOnFail: true,
+        ...options,
+      },
+      groupId
+    );
```

Comment:
why we are not using the `add` function from this class?

Reply:
At this state, we could not call this.add because for StandardQueueService it is not implemented.

However, due to other comments you made I will remove the addMinimalJob from base.